packages/@tinacms/cli/src/cmds/compile/defaultSchema.ts

```diff
@@ -34,6 +34,12 @@ const schema = defineSchema({
       name: 'post',
       path: 'content/posts',
       format: 'mdx',
+      ui: {
+        router: ({ document }) => {
+          // This can be used to add contextual editing to your site. See https://tina.io/docs/tinacms-context/#accessing-contextual-editing-from-the-cms for more information.
+          return undefined
```

Comment:
Wouldn't we want to send them to the demo page?

Reply:
Yeah that would make sense! I will update